### ISSUE TITLE
SQL Error from withOrdering() when trying to sort a relationship count

### ISSUE THREAD

**Issue by Phr33d0m** [laravel/nova-issues#1410]
- Laravel Version: 5.8.2
- Nova Version: 2.0
- PHP Version: 7.2.15-1+ubuntu18.04.1+deb.sury.org+1
- Operating System and Version: ---doesnotapply---
- Browser type and version: ---doesnotapply---

### Description:
Hi,
```LensRequest``` provides a nice utility called ```withOrdering()```. The problem is that it **always** appends the table name, as can be seen by the function's code here:

```php
return $query->orderBy(
    $query->getModel()->getTable().'.'.$this->orderBy,
    $this->orderByDirection === 'asc' ? 'asc' : 'desc'
);
```

Which breaks if you're trying to sort a subquery, for example a count on a relationship.

### Steps To Reproduce:

1. Author -> hasMany Book
2. 
```php
return $request->withOrdering($request->withFilters(
	$query->withCount('books')
));
```
3. 
```php
Number::make('Books', 'books_count')->sortable()
```
4. Click on the table column to sort by books count.
5. You'll get an error message that ```[table].[books_count]``` does not exist which is obvious.

In the meantime I've completely ignored the ```withOrdering()``` goodie and replaced it with:
```php
$query->withCount('books');

if($request->orderBy AND $request->orderByDirection) {
	$query->orderByRaw($request->orderBy . ' ' . $request->orderByDirection === 'asc' ? 'asc' : 'desc');
}

return $request->withFilters($query);
```

**Comment by rdarcy1** [comment]
I think this is a duplicate of #726?

**Comment by jbrooksuk** [comment]
Duplicate of #726.

**Comment by tainmar** [comment]
But it is still an issue @jbrooksuk and you have closed the duplicate #726
